skbeam/core/tests/test_recip.py

```diff
@@ -136,6 +138,30 @@ def test_hkl_to_q():
     npt.assert_array_almost_equal(b_norm, recip.hkl_to_q(b))
 
 
+def test_gisaxs():
+    incident_beam = (1.0, 1.0)
+    reflected_beam = (3.0, 3.0)
+    pixel_size = (1.0, 1.0)
+    detector_size = (5, 5)
+    dist_sample = 5.0
+    wavelength = 2*np.pi*0.01
+    theta_i = 0.0
+
+    (alpha_i, theta_f, alpha_f, tilt_angle,
+     qx, qy, qz, qr) = recip.gisaxs(incident_beam, reflected_beam, pixel_size,
+                                    detector_size, dist_sample, wavelength,
+                                    theta_i=0.0)
+    assert_array_almost_equal(0.78539816339744828, tilt_angle, decimal=8)
+    assert_array_almost_equal(2*10**(-7), alpha_i, decimal=8)
+    assert_array_almost_equal(np.array([-1.00000000e-07, 0.00000000e+00,
+                                        1.00000000e-07, 2.00000000e-07,
+                                        3.00000000e-07]), theta_f[1, :],
+                              decimal=8)
+    assert_array_almost_equal(np.array([-4.00000000e-07, -2.00000000e-07,
+                                        7.99387344e-21, 2.00000000e-07,
+                                        4.00000000e-07]), alpha_f[:, 1])
```

Comment:
Can you rewrite these last two assertions as:
```python
    theta_f_target = 10**-7 * np.array([-1., 0., 1., 2., 3.])
    alpha_f_target = 10**-7 * np.array([-4, -2, 0, 2, 4])
    assert_array_almost_equal(theta_f_target, theta_f[1, :])
    assert_array_almost_equal(alpha_f_target, alpha_f[:, 1])
```

I think that this will help with the readability of this test